test.properties

```diff
@@ -4986,6 +4987,9 @@
     test.batch.run.property.query[functional-tomcat-mysql80-jdk21_zulu][portal-release][portal]=\
         (portal.smoke == true)
 
+    test.batch.run.property.query[functional-tomcat-mysql84-jdk21_zulu][portal-release][portal]=\
+        (portal.smoke == true)
+
```

Comment:
Will need to add functional-tomcat-mysql84-jdk21_zulu to the portal-release batch names